x-pack/plugins/ml/public/application/util/index_utils.ts

```diff
@@ -129,6 +129,6 @@ export function timeBasedIndexCheck(dataView: DataView, showNotification = false
  * Returns true if the data view index pattern contains a :
  * which means it is cross-cluster
  */
-export function isCcsIndexPattern(dataViewIndexPattern: string) {
-  return dataViewIndexPattern.includes(':');
+export function isCcsEsIndexPattern(esIndexPattern: string) {
```

Comment:
In some situations where the variable is a string I think "index pattern" is correct.
In the DataView class they still refer to the string which matches indices as the "index pattern"
https://github.com/elastic/kibana/blob/426a379f311fe76fccfe1fbf260ca0ea1b41f772/src/plugins/data_views/common/data_views/abstract_data_views.ts#L200-L201